components/mailer.rst

```diff
@@ -177,6 +177,43 @@ the application or when using a self-signed certificate::
 
     The ``verify_peer`` option was introduced in Symfony 5.1.
 
+Other options
+-------------
+
+.. versionadded:: 5.2
+
+    These options were added in Symfony 5.2.
+
+command
+~~~~~~~
+
+Command to be executed by ``sendmail`` transport.
+
+    $dsn = 'sendmail://default?command=/usr/sbin/sendmail%20-oi%20-t'
+
+local_domain
+~~~~~~~~~~~~
+
+The domain name to use in ``HELO`` command.
```

Comment:
```suggestion
The domain name to use in ``HELO`` command::
```

+

Can you explain to me what ``HELO`` is? Maybe we could add a link to a wikipedia page or sth. else?

Reply:
You mean like https://en.wikipedia.org/wiki/Simple_Mail_Transfer_Protocol#SMTP_transport_example ? That's were "HELO" links to from other pages on Wikipedia at least.